ledger-tool/src/main.rs

```diff
@@ -1681,15 +1689,6 @@ fn main() {
                     .requires("accounts")
                     .help("Do not print account data when printing account contents."),
             )
-            .arg(
-                Arg::with_name("encoding")
-                    .long("encoding")
-                    .takes_value(true)
-                    .possible_values(&["base58", "base64", "base64+zstd", "jsonParsed"])
-                    .default_value("base58")
-                    .requires("accounts")
```

Comment:
Having `requires("accounts")` was useless here, as this arg was being added to the `accounts` subcommand so `accounts` being specified is implied.

Reply:
this one's for `solana-ledger-tool genesis`, which has a `--accounts`.

https://github.com/solana-labs/solana/pull/28490/files#diff-fe20bb9062954767913766205f72e8eb5ae88909c0d2529ec706e27a3e0eb800L1672

Reply:
D'oh, you're right, sorry for the false alarm. Fired this PR off without thinking it through

Reply:
you can still dereplicode it.  just `&accounts_data_encoding_arg.requires("accounts")` here

Reply:
Compiler says no
```
error[E0382]: borrow of moved value: `accounts_data_encoding_arg`
    --> ledger-tool/src/main.rs:2025:18
     |
1494 |     let accounts_data_encoding_arg = Arg::with_name("encoding")
     |         -------------------------- move occurs because `accounts_data_encoding_arg` has type `Arg<'_, '_>`, which does not implement the `Copy` trait
...
1684 |             .arg(&accounts_data_encoding_arg.requires("accounts"))
     |                                              -------------------- `accounts_data_encoding_arg` moved due to this method call
...
2025 |             .arg(&accounts_data_encoding_arg)
     |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^ value borrowed here after move
     |
note: this function takes ownership of the receiver `self`, which moves `accounts_data_encoding_arg`
```
I guess I could throw a `clone()` in there to avoid the borrow-after-move:
```rust
.arg(&accounts_data_encoding_arg.clone().requires("accounts"))
```